wallet-new/test/Integration/Fixtures.hs

```diff
@@ -0,0 +1,20 @@
+module Integration.Fixtures (MonadIntegrationFixtures(..)) where
+
+import           Universum
+
+import           Cardano.Wallet.Client.Http
+
+import           Integration.Util (asksM, fromResp)
+
+import qualified Prelude
+
+
+class MonadIntegrationFixtures m where
+    getArbitraryAccount :: m (Account, Wallet)
+
+instance MonadIntegrationFixtures (ReaderT (WalletClient IO) IO) where
+    getArbitraryAccount = do
+        wallets <- asksM getWallets >>= fromResp
+        let wallet@Wallet{..} = Prelude.head wallets
+        accounts <- asksM (`getAccounts` walId) >>= fromResp
+        return (Prelude.head accounts, wallet)
```

Comment:
Some concept as in `Integration/Assertions`. We define here a DSL for prepatory actions we need to execute in our specs. The main idea is really to keep the actual spec really readable (see below) and factor out the noise. Note that the implementation here is a bit dumb as it re-fetch content from the server all the time. However, since this is simply running against a demo cluster during testing, that's a fairly acceptable cost.

Reply:
I'm not sure we should be programming this in `mtl` style.

Reply:
Are you referring to the `askM` helper function, or more generally, to the approach using a `MonadFixture` and a `MonadAssertion` to express integration specs?